qiskit/pulse/library/parametric_pulses.py

```diff
@@ -447,3 +486,10 @@ def __init__(self,
         """
         super().__init__(duration, amp, name)
         warnings.warn("The ConstantPulse is deprecated. Use Constant instead", DeprecationWarning)
+
+
+def is_parameterized(value: Any) -> bool:
```

Comment:
Should this be private to the module?
```suggestion
def _is_parameterized(value: Any) -> bool:
```

Reply:
yes thank you!